transforms/__testfixtures__/radium-to-glamor/this-props-spread.input.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Radium from "radium";
+import OtherComponent from "./OtherComponent";
+
+const Spreader = React.createClass({
+
+    render() {
+        const { ComponentClass, OtherClass } = this.props; // eslint-disable-line react/prop-types
+
+        return (
+            <ComponentClass
+                {...this.props}
+                style={this.getStyles()}
+            >
+                <div
+                    {...this.props}
+                    style={this.getDivStyles()}
+                >
+                    DANGER BECAUSE DOM
+                </div>
+                <OtherComponent
+                    {...this.props}
+                    style={this.getDivStyles()}
+                >
+                    NO DANGER - NO MESSAGE
+                </OtherComponent>
+                <li
+                    {...this.props}
+                    otherProp={true}
+                >
+                  NO DANGER BECAUSE NO STYLE
```

Comment:
It's not necessarily the case that there's no danger when there's not style props getting passed to the DOM. It can still be part of the spread props.